deploy/kubernetes/driver/kubernetes/manifests/node-server.yaml

```diff
@@ -171,3 +173,7 @@ spec:
         - name: customer-auth # altough its not required, This is just to finish lib configuration which is a common code in the driver
           secret:
             secretName: storage-secret-store
+        - hostPath:
+            path: /var/lib/kubelet/
+            type: Directory
```

Comment:
If we use this on IKS (ubuntu) will this have any impact ?

Reply:
@sameshai  Tested on IKS and works as expected